From 7ceea548528bc983e77f2fbd1395e2d0acded8df Mon Sep 17 00:00:00 2001
From: Zhixiong Chi <zhixiong.chi@windriver.com>
Date: Mon, 13 Apr 2020 05:18:05 -0700
Subject: [PATCH 5/5] arm64: dts: qm-mek/qxp-mek: Suppress DTC warnings related
 to max9286_mipi@6A

This patch suppresses DTC warnings which are related to LVDS like below:
arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-mipi-ov5640.dtb:
Warning (graph_endpoint): /i2c@58226000/max9286_mipi@6A/port/endpoint:
graph connection to node '/camera/csi@58227000/port@0/endpoint'
is not bidirectional

Signed-off-by: Zhixiong Chi <zhixiong.chi@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-ov5640.dts  | 2 ++
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-mipi-ov5640.dts | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-ov5640.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-ov5640.dts
index df8bf72b05bc..8983c21a1004 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-ov5640.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-mipi-ov5640.dts
@@ -14,6 +14,8 @@
 
 #include "fsl-imx8qm-mek.dts"
 
+/delete-node/ &max9286_0_ep;
+
 &iomuxc {
 	imx8qm-mek {
 		pinctrl_mipi_csi0: mipicsi0grp{
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-mipi-ov5640.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-mipi-ov5640.dts
index 7756d686caf4..cd69ce49377b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-mipi-ov5640.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-mipi-ov5640.dts
@@ -15,6 +15,8 @@
 #include "fsl-imx8qxp-mek.dts"
 #include "fsl-imx8qxp-mek-lvds0-it6263.dtsi"
 
+/delete-node/ &max9286_0_ep;
+
 &iomuxc {
 	imx8qxp-mek {
 		pinctrl_mipi_csi0: mipicsi0grp{
-- 
2.17.1

